src/Hydration-repo.js

```diff
@@ -0,0 +1,13 @@
+import Hydration from './Hydration';
```

Comment:
I really like the look of all of these new class repo instances. Great work cleaning all that up